src/Integration.UnitTests/Binding/SolutionBindingOperationTests.cs

```diff
@@ -318,7 +319,7 @@ public void SolutionBindingOperation_CommitSolutionBinding()
             };
 
             // Sanity
-            this.solutionBinding.AssertWriteSolutionBindingRequests(0);
+            this.solutionBinding.AssertWrittenFiles(0);
```

Comment:
There's no SCC is internal to SolutionBinding so can't very pending vs. written, only written at the unit test level.